app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+import os
```

Comment:
Consider adding a docstring to the `move_file` function to describe its purpose, parameters, and any exceptions it might raise. This will improve code readability and maintainability.